### PR TITLE
Update TranslatorText.json

### PR DESCRIPTION
**Pull request by swmachan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json [hunk 1]
```diff
@@ -603,17 +639,19 @@
...
-            }
-          }
+            "name": "Request Body",
+            "description": "\n  The body of the request is a JSON array. Each array element is a JSON object with a string property named `Text`, which represents the string to convert.\n  The following limitations apply:\n  * The array can have at most 10 elements.\n  * The text value of an array element cannot exceed 1,000 characters including spaces.\n  * The entire text included in the request cannot exceed 5,000 characters including spaces.\n",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Respnse Body",
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]:
```suggestion
            "name": "Response Body",
```

Comment (swmachan) [reply specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json]: Fixed the typo in next iteration

**Review by yangyuan — APPROVED** [review]

**Comment by akning-ms** [comment]
Hi @swmachan, is this PR still need as it is idle for a long time

**Comment by swmachan** [comment]
akning-ms - no this PR is not needed.

**Pull request closed without merging**